lib/improver/nbhood/circular_kernel.py

```diff
@@ -41,6 +46,47 @@
 MAX_RADIUS_IN_GRID_CELLS = 500
 
 
+def circular_kernel(fullranges, ranges, weighted_mode):
+    """
+
+    Method to create a circular kernel.
+
+    Parameters
+    ----------
+    fullranges : Numpy.array
+        Number of grid cells in all dimensions used to create the kernel.
+        This should have the value 0 for any dimension other than x and y.
+    ranges : Tuple
+        Number of grid cells in the x and y direction used to create
+        the kernel.
+
+    Returns
+    -------
+    kernel : Numpy.array
+        Array containing the circular smoothing kernel.
+        This will have the same number of dimensions as fullranges.
+
+    """
+    # Define the size of the kernel based on the number of grid cells
+    # contained within the desired radius.
+    kernel = np.ones([int(1 + x * 2) for x in fullranges])
+    # Create an open multi-dimensional meshgrid.
+    open_grid = np.array(np.ogrid[tuple([slice(-x, x+1) for x in ranges])])
```

Comment:
Why do you need tuple() here?

Reply:
`tuple()` isn't required. I've removed this.